### ISSUE TITLE
[SideNavigation][A11y]: UI element type is not read by screenreader

### ISSUE THREAD

**Issue by volha-arkhipenka-sap** [SAP/ui5-webcomponents#10985]
### Bug Description

With Voiceover enabled, the UI element type for `SideNavigationItem` is not read.

### Affected Component

https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/layouts-floorplans-sidenavigation--default

### Expected Behaviour

UI element type (such as `button`, `menu item`, etc.) should be communicated to the user.

### Priority

Medium

### UI5 Web Components Version

2.7.0

### Browser

Safari

### Operating System

iOS 18.1.1

### Additional Context

The issue is also reproducible on macOS Sequoia 15.3.1 for Google Chrome Version 133.0.6943.127 and Safari Version 18.3.

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rd ,
could you please check this issue?
The issue reproduces following the described steps.

Thanks,
Todor

**Comment by GerganaKremenska** [comment]
Hello @volha-arkhipenka-sap, 

This seems to be a vendor issue or expected behavior since side navigation is following all aria best practices and it is implented following this [example](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/).
The issue is reproducable in the best practice pages also, so i am closing the issue.

Best Regards,
Gergana

**Comment by dreamt12** [comment]
Hi, @GerganaKremenska,
I'm not sure I understand how this is a vendor issue? Which vendor? I see this has been tested over 2 devices with multiple browsers.
Also when you say that the issue is reproduced in best practices as well - what does it mean? Best practices didn't consider this case?
Best practices need to be updated so this case would actually work?

If the aforementioned case is used, and it's valid, don't you agree that improvement to the component is in order?

**Comment by GerganaKremenska** [comment]
> Hi, [@GerganaKremenska](https://github.com/GerganaKremenska), I'm not sure I understand how this is a vendor issue? Which vendor? I see this has been tested over 2 devices with multiple browsers. Also when you say that the issue is reproduced in best practices as well - what does it mean? Best practices didn't consider this case? Best practices need to be updated so this case would actually work?
> 
> If the aforementioned case is used, and it's valid, don't you agree that improvement to the component is in order?

Hello @dreamt12, 

By vendor i mean the screen reader. By which i mean Voice Over.

The reported issue is on one browser, one device, since it is not described in the issue i do not have any othere information.

When something is reproducable in the best practice example it proves that the issue is not in the ui5 web components, beacuse it is reproduced outside of the freamowork with native html. It is not a "case to consider" if a "type"/role of an element is not announced, the sample follows the aria standard and if a role is not announced its not something a sample(best practise example) should consider, maybe it should not be announced to begin with. 
Best practice exampe does not need to be updated if screen reader is misbehaving in some way.

The component does not need an improvment as correct roles are used, as far that the screen reader does not read something out as some user expects its a question for the vendor/screen reader. I have tested with NVDA, Voice over and JAWS and only jaws announces that it is entering a tree view, and after that it announces the elements texts, which seems correct, no roles (e.g treeitem). 


Best Regards,
Gergana

**Comment by GerganaKremenska** [comment]
Hello @dreamt12,

I my own opinoin in best position is the preson that find the issue to reported it, becasue they have the exmaple and steps to reproduce and can keep trck of it when it is updated. In this case i do not think this is an issue, but an expected behavior after checking with two othere screen readers.  

Best Regards,
Gergana
